Source/ACE.Server/Physics/Entity/GfxObjCache.cs

```diff
@@ -8,8 +8,7 @@ namespace ACE.Server.Physics.Entity
 {
     public static class GfxObjCache
     {
-        //public static readonly ConcurrentDictionary<uint, GfxObj> GfxObjs = new ConcurrentDictionary<uint, GfxObj>();
-        public static readonly ConcurrentDictionary<uint, WeakReference<GfxObj>> GfxObjs = new ConcurrentDictionary<uint, WeakReference<GfxObj>>();
+        public static readonly ConcurrentDictionary<uint, GfxObj> GfxObjs = new ConcurrentDictionary<uint, GfxObj>();
```

Comment:
the caches are designed to ensure there are not multiple instances to the same resource (similar to DatLoader)

however, not similar to DatLoader, they are not intended to keep resources permanently loaded throughout the lifetime of the server. if i am running a smaller server for multiple hours, i will most likely only have a small subset of these resources loaded at any point. i do not want my computer to continually permaload more resources as it comes across them.

the physics caches do not work like the DatLoader in this way, or even the Weenie cache. if there should be a similar option to the Weenie cache to preload all of these resources on startup, and keep them permaloaded throughout the lifetime of the server, then that should be a configurable option. (a lot of the larger servers also run with weenie preloading disabled)

Reply:
I can make this optional, but when we tested this, it was tens of MB's we're talking about.

If you want it configurable for 10's of MB's, let me know and I'll make the change

Reply:
yeah for GfxObjCache in particular, might not add up to much... was making more of a general statement of how the physics entity caches differ from other things like DatLoader / WeenieCache